### ISSUE TITLE
Allow searching through created lists

### ISSUE THREAD

**Issue by Stefan240** [bookwyrm-social/bookwyrm#2823]
First thank you to everyone who contributed to this and made this possible. 

I just opened my instance and added all the books that I could not find yet. 

My proposal goes into a similar direction as #2079 I would like to add another search. I created a big list of books of a specific type and I really would like to use that list as a small "database" of sorts, so I can link this list to people and they see all the books in that particular style of books. 

Would it be possible to add a search bar to the list so a user can search for something specific inside the list only?

**Comment by jaschaurbach** [comment]
So you want kind of local search only, right?

**Comment by mouse-reeve** [comment]
I think this is similar to the ticket about searching "your books" and would be very doable -- rather than being a general search, it would be a query within the books in a specific list. Does that sound right?

**Comment by Stefan240** [comment]
> I think this is similar to the ticket about searching "your books" and would be very doable -- rather than being a general search, it would be a query within the books in a specific list. Does that sound right?

Yes this exactly was the idea, which is why I linked the issue where this was discussed. 


> So you want kind of local search only, right?

There is really no need for a local only search really. Bookworm does a local search when you search something by default. At least from what I have seen, when searching on my instance.
